pallets/verifiers/src/tests.rs

```diff
@@ -95,32 +137,164 @@ mod register_should {
         assert_eq!(info.pays_fee, Pays::Yes);
         assert_eq!(info.weight, MockWeightInfo::register_vk(&43));
     }
-}
 
-pub mod submit_proof_should {
-    use hex_literal::hex;
+    #[rstest]
+    fn hold_a_deposit(mut test_ext: sp_io::TestExternalities) {
+        test_ext.execute_with(|| {
+            let initial_reserved_balance = Balances::reserved_balance(USER_1);
+            assert_ok!(FakeVerifierPallet::register_vk(
+                RuntimeOrigin::signed(USER_1),
+                Box::new(42)
+            ));
+            assert!(Balances::reserved_balance(USER_1) > initial_reserved_balance);
```

Comment:
If one wants to be pedantic, you could compute the actual hold amount and check that indeed the user's account has been emptied by that exact amount.

Reply:
Addressed in https://github.com/zkVerify/zkVerify/pull/178/commits/a4eeb34edcd2f58693e2b957c85ec532be4915e0